mscore/musescore.cpp

```diff
@@ -1069,13 +1069,18 @@ MuseScore::MuseScore()
       connect(ag, SIGNAL(triggered(QAction*)), SLOT(cmd(QAction*)));
 
       mainWindow = new QSplitter;
+      mainWindow->setObjectName("mainwindow");
+      mainWindow->setAccessibleName("main window");
```

Comment:
are these texts the screen reader reads? If so they should be translatable (and start with upper case)

Reply:
Good point.  Under normal circumstances the screen won't read most of these; they are more for the benefit of people writing JAWS scripts and as such might be better *not* translated.  But some of them can indeed be read.  I need to learn more about how the scripting works, but hopefully we can get the scripts to work from the object name alone.